### ISSUE TITLE
no pcap for 1.0.1

### ISSUE THREAD

**Issue by surskitt** [yellows8/3ds_smashbroshax#8]
With the new 1.1.3 update in the estore, the only option for those on an unsupported version is to delete our update and downgrade to 1.0.1 (EU estore download version). However there is no pcap file for 1.0.1.

Is there any way that those of in this situation can continue to use smashhax?

**Comment by yellows8** [comment]
Uhhh, AFAIK 1.0.1 is JPN-only?

**Comment by ihaveamac** [comment]
my downloaded version of SSB4 is 1.0.1 when I delete/disable the update data. USA console

**Comment by adamasstokhorst** [comment]
seconding request for 1.0.1 pcap file, I removed update data and it went back to 1.0.1 (US version) and so far, none of the gameusa pcaps work.

**Comment by EverOddish** [comment]
I also have 1.0.1 US version, is there any way you can add support for it?

**Comment by scojasp** [comment]
Adding onto the request as well; US cart is version 1.0.1 when playing the game cart without any updates/on a blank 3DS.

**Comment by UseAfterMe** [comment]
Physical cartridge ver 1.1.0 US
Crashes to main menu and forces system reboot.
Tested with eshop ver 1.0.0 exploit runs perfectly and launches HBL
This has to be an issue with either the pcap or they patched physical copies of 1.1.0
Please chime in and let us know, because quite a few people are having the same issue with physical cartridge 1.1.0 

Thanx

**Comment by yellows8** [comment]
"or they patched physical copies of 1.1.0" Obviously it wouldn't crash at all in a vuln-fix build......

And of course all I have for v1.1.0 is the update-title.

**Comment by UseAfterMe** [comment]
I figured as much on the physical copy of 1.1.0
Some thing must be different from the update to the physical copy I guess.
Maybe different offset?
The game does crash but its like its not finding the payload to run and just crashes the entire system.

**Comment by viewtifulv** [comment]
The physical copy I got is also 1.0.1. Is there any chance that a pcap will be added for it?

**Comment by warchicken** [comment]
Pcap for 1.0.1 is it possible? Since browserhax is gone this is my only hope of getting to home brew a downgrade( I have everything in SD ready for downgrade).. I've been trying tirelessly to fix what my little kid did out of curiosity when he updated my downgraded 9.2sysnand to 10.7, .. I lost everything I waited so much for in 3 seconds.. Since I was originally in 10.5.. I've been trying smashhax for 4 days straight.. Is there a way to add support for pcap 1.0.1 , .. I need home brew and I can't have any of the other exploits, please help

**Comment by naendow** [comment]
So, is there anyone that could help us? There are a few persons that are just able to use the 1.0.1 and where SmashBrosHax would be the onliest entrypoint. It would be really nice if someone could do this work for us. :+1:

**Comment by warchicken** [comment]
https://mega.nz/#!GB5gQaBb!guv6ULGRqhnUWw9DuRzmnU1wcmLStfsqFxc0Gclbv0A hope
this works let me know

**Comment by naendow** [comment]
That is a not working link. :(

**Comment by warchicken** [comment]
Well I downloaded it from a Windows based laptop and the link was working

**Comment by naendow** [comment]
Could you reupload it and post the link here?

Do you have a european or a us-ssb?

**Comment by warchicken** [comment]
i found the link in gbatemp forum

**Comment by aaronson2012** [comment]
check my issue for working 1.0.1 pcap

**Comment by prayerie** [comment]
anyone got the link/can reupload the file?

**Comment by aaronson2012** [comment]
https://onedrive.live.com/redir?resid=A30F7E100D513D7D!10427&authkey=!AE_QvKI6-RIaBQE&ithint=file%2cpcap

**Comment by Amirrasa** [comment]
I need to use smashhax with my 1.0.1 EU cartridge, the thing I don't understand is if 1.0.1 EU/Jap is not supported at all or I need to compile it?

**Comment by prayerie** [comment]
@Amirrasa it's definitely supported - it just needs to be compiled. 
@aaronson2012 thanks for the link, however I was looking for gameother (eu/jp)

**Comment by Amirrasa** [comment]
@minecast I saw a post by someone who said she could find a pcap that worked on her v1.0.1 Japanese eshop copy but I had no luck with it on my 1.0.1 European card. Here is the link:
https://gbatemp.net/threads/tutorial-using-smashhax-with-linux.397194/page-15#post-6227641

**Comment by Quin452** [comment]
If, going by the files, the pcaps are either US or OTHER.

I've managed to find a [1.0.1 OTHER pcap](https://www.dropbox.com/s/u4j19jivqvem4kf/smashbros_gameotherv101_beaconhax.pcap?dl=0), but not too sure if it is legit.  I renamed and added the file to the folder, and ran the beacon.  It sends out fine, but still crashes and reboots.

I cannot figure out what is the cause -.-

**Comment by nathyong** [comment]
I just tested out that 1.0.1 OTHER pcap, on 10.7.0-32E sysnand, and it works for me as long as I enter and exit the home menu just before pressing A to enter the "group" menu in Smash Bros. I am running an Australian version of Smash Bros 1.0.1, which should be identical to the European version.

If I don't switch between the home menu then it takes a while to activate, and crashes most of the time.

**Comment by MarreroJorge** [comment]
any news on working 1.0.1 us PCAP? ive tried other on us version and it gets me stuck on the red screen

**Comment by yellows8** [comment]
@DrWhiteRaven At that point *hax payload is already running..... Extract starter-kit to SD.

**Comment by MarreroJorge** [comment]
Its in the root of the sd : \ 
I get the same result with it there or not there.

**Comment by yellows8** [comment]
The contents are supposed to be extracted to the root, there should be boot.3dsx  @ root.

**Comment by MarreroJorge** [comment]
placed boot.3dsx @root do i have to do anything with 3ds folder? now it pauses at "ropbin menu is ready"

**Comment by yellows8** [comment]
Just try again.

**Comment by MarreroJorge** [comment]
ok, ill keep trying.
UPDATE: Thank you! got it to work after a couple more tries.

**Comment by taxon221** [comment]
So this should be closed now right?

**Comment by mimaqua** [comment]
Got the 1.0.1 OTHER pcap posted by Quin452. Worked on EUR cartridge :)

All the helper files did not work so I had to follow the step by step guide on gbatemp. My laptop network card couldn't send the hack, I had to use a usb wi-fi dongle instead. 

I added a few extra steps:
I did this after I patched the downloaded aircrack.  
1. install aircrack-ng (sudo apt-get install aircrack-ng)
2. start monitor mode with aircrack, replace YourDongleHere with your card/dongle name from ip link (sudo airmon-ng start YourDongleHere)
3. The dongle will eventually be renamed (mine was renamed to wlan0mon)
4. Now you have your dongle in a monitor interface and can set this one to monitor mode with the commands from the guide and then send the beacon

I don't know if all the steps are necessary but it worked for me so I hope this is worth sharing :)
